graylog2-web-interface/src/components/event-notifications/event-notification-form/EventNotificationForm.jsx

```diff
@@ -19,6 +23,8 @@ class EventNotificationForm extends React.Component {
     onChange: PropTypes.func.isRequired,
     onCancel: PropTypes.func.isRequired,
     onSubmit: PropTypes.func.isRequired,
+    onTest: PropTypes.func.isRequired,
+    testState: PropTypes.object.isRequired,
```

Comment:
This is a required PropType but it's never set where the component is used.

Reply:
That's what the store provides for us.